overlays/extensions/power_optimize_brightness.vspec

```diff
@@ -0,0 +1,33 @@
+#
```

Comment:
This is an overlay example corresponding to current Mobis proposal. Shall possibly not be be part of final PR as the solution might be to project/platform/OEM-specific